phpBB/adm/style/permissions.js

```diff
@@ -279,6 +279,16 @@ function reset_role(id) {
 	}
 
 	t.options[0].selected = true;
+
+	(function($)// Avoid conflicts with other libraries
```

Comment:
why are you using a sub function? just rewrite the whole function for jquery...Also please remove the conflict comments they are redundant

Reply:
1. change `var t =` line to `var $t = $(id)`
2. `var parent = $(t).parent();` should just be `$t.parent();`
3. everywhere t is used change it to `$t`
4. wrap the whole reset_role function than in the self-invoking function.

Reply:
@hanakin Thanks, but that kinda makes no sense for me to try blindly executing this instruction.
That simply leads me to facing errors like
```
permissions.js:271 Uncaught ReferenceError: jQuery is not defined
    at permissions.js:271
(anonymous) @ permissions.js:271
Uncaught ReferenceError: reset_role is not defined
    at HTMLAnchorElement.onclick 
Uncaught ReferenceError: reset_role is not defined
    at HTMLAnchorElement.onclick 
```

Where I have no clue about fixing them.
I guess it's easier to close the PR and leave it as is.

Reply:
hmm not sure why you did not get the error before then. Its saying that jquery is not loaded prior to the loading of permissions.js